ddtrace/ext/ci.py

```diff
@@ -295,14 +304,29 @@ def extract_circle_ci(env):
         JOB_NAME: env.get("CIRCLE_JOB"),
         PROVIDER_NAME: "circleci",
         WORKSPACE_PATH: env.get("CIRCLE_WORKING_DIRECTORY"),
-        # OrderedDict is necessary for comparing against a fixture in testing
         _CI_ENV_VARS: json.dumps(
-            OrderedDict(
-                [
-                    ("CIRCLE_WORKFLOW_ID", env.get("CIRCLE_WORKFLOW_ID")),
-                    ("CIRCLE_BUILD_NUM", env.get("CIRCLE_BUILD_NUM")),
-                ]
-            ),
+            {
+                "CIRCLE_WORKFLOW_ID": env.get("CIRCLE_WORKFLOW_ID"),
+                "CIRCLE_BUILD_NUM": env.get("CIRCLE_BUILD_NUM"),
+            },
+            separators=(",", ":"),
+        ),
+    }
+
+
+def extract_codefresh(env):
+    # type: (MutableMapping[str, str]) -> Dict[str, Optional[str]]
+    """Extract CI tags from Codefresh environ."""
+    build_id = env.get("CF_BUILD_ID")
+    return {
+        git.BRANCH: env.get("CF_BRANCH"),
+        PIPELINE_ID: build_id,
+        PIPELINE_NAME: env.get("CF_PIPELINE_NAME"),
+        PIPELINE_URL: "https://g.codefresh.io/build/{0}".format(build_id),
```

Comment:
Is this something other than `CF_BUILD_URL` specified in https://codefresh.io/docs/docs/pipelines/variables/ ?

If it is, is there a way we can use `CF_URL` instead of hardcoding `g.codefresh.io` ?